src/app/features/create-event/create-event/create-event.component.ts

```diff
@@ -1,9 +1,16 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component, inject, OnDestroy } from '@angular/core';
+import { CreateEventFormService } from '../create-event-form.service';
 
 @Component({
   selector: 'app-create-event',
   templateUrl: './create-event.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
   providers: [],
 })
-export class CreateEventComponent {}
+export class CreateEventComponent implements OnDestroy {
+  private createEventFormService = inject(CreateEventFormService);
+
+  ngOnDestroy() {
+    this.createEventFormService.getForm().reset();
```

Comment:
niech to będzie po prostu metoda reset, a implementacje czy `getForm().reset()` będzie już zamknięta w serwisie. Wtedy też bedzie trzeba prawdopdobnie stworzyć nowy formularz dla pewności, że nie ma żadnej zmiany ze starego